src/util/asmap.cpp

```diff
@@ -77,9 +77,10 @@ uint32_t Interpret(const std::vector<bool> &asmap, const std::vector<bool> &ip)
             return DecodeASN(pos, endpos);
         } else if (opcode == 1) {
             jump = DecodeJump(pos, endpos);
+            assert(pos + jump > pos);
```

Comment:
Why this assert is needed here?

Reply:
I mean for that to protect against the unlikely case of overflow of `pos + jump` overflowing - if an overflow occurs, `pos + jump` will be less than `pos`.

Reply:
Is that condition impossible? If it is possible to reach I'm not certain aborting is the right thing to do :)

Reply:
I don't think this should be an assert. It can be caused by incorrect input (an corrupt `asmap` file), not just developer errors. Don't we have other ways of reporting an error here?

Reply:
Updated to `break` which results in returning `0`, which is not a valid ASN.